packages/rocketchat_theme/client/imports/components/header.css

```diff
@@ -34,9 +34,7 @@
 	&__block {
 		display: flex;
 
-		margin: 0 -0.5rem;
-
-		padding: 0 0.5rem;
+		margin: 0 1rem 0 0;
```

Comment:
We avoid unidirectional margins and paddings -- in this case, margin to the right only -- since they don't work well in right-to-left languages. Try to use the same margins on both sides and test them against languages like Arabic.

Reply:
@tassoevan putting margin on left will make it not aligned with the rest of the content

Reply:
We often use the technique of negative margins on parent to handle this.